tests/mesh/test-hausdorff-distance.cpp

```diff
@@ -0,0 +1,29 @@
+#include <geode/basic/assert.h>
+#include <geode/basic/logger.h>
+
+#include <geode/geometry/aabb.h>
+#include <geode/geometry/point.h>
+
+#include <geode/mesh/core/triangulated_surface.h>
+#include <geode/mesh/helpers/aabb_surface_helpers.h>
+#include <geode/mesh/helpers/hausdorff_distance.h>
+#include <geode/mesh/io/triangulated_surface_input.h>
+
+#include <geode/tests/common.h>
+
+void test()
+{
+    const std::string filename( "../tests/data/iter1.og_tsf3d" );
```

Comment:
use this https://github.com/Geode-solutions/OpenGeode/blob/master/tests/mesh/test-graph.cpp#L168